### ISSUE TITLE
Roles

### ISSUE THREAD

**Issue by Sadisticsmile** [fetlife/glossary#19]
You all have everything imaginble an then some but Couple and i see more couples than any other role plus the andriod app has been out for years what about apple users?

**Comment by APTememe** [comment]
I agree, need more couple roles for couples who want to have a joint profile.
